code/heightofbt.java

```diff
@@ -0,0 +1,110 @@
+package practiceds;

```

Comment:
This code shall be removed from the code directory 👍 

The above file is the same one 👍